### DISCUSSION TITLE
case-studies/nike/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#420] [Website Discussions]
# case-studies/nike/

Our friends at Orange Studios recently collaborated with Nike to do an incredible large format installation for the Olympic Games - all cut in Final...

https://fcp.cafe/case-studies/nike/

**Comment by florian-duffe** [comment]
So great, thanks Oli! I'm curious about the notch codec. Does this run natively on FCP? Did you have to transcode? Is this particularly for graphics? What are your experiences with it?

**Reply by latenitefilms** [reply]
Unfortunately not. I believe he was using Adobe Media Encoder to do the Notch encoding. Hopefully one day Notch will create a [Encoder Extension for Compressor](https://developer.apple.com/documentation/professional-video-applications/encoder-extensions).
